CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Prowide ISO 20022 - CHANGELOG
- 
+
+#### 9.5.6 - January 2025
```

Markdownlint (0.37.0)

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)